apps/end-tenancy/acceptance/features/confirm.js

```diff
@@ -77,14 +37,3 @@ Scenario('I see the correct table information if I am a landlord checking a sing
   yield I.refreshPage();
   confirmPage.checkData('check-landlord');
 });
-
-Scenario('I am taken to the confirmation page if I accept the declaration', function *(
```

Comment:
this test is going to be redundant as a result of this PR https://github.com/UKHomeOffice/end-tenancy/pull/84